docs/pages/setup/helm-deployments/aws.mdx

```diff
@@ -530,6 +532,53 @@ Here's an example where we set the chart to use 3 replicas:
   To change `chartMode`, `clusterName`, or any `aws` settings, you must first uninstall the existing chart and then install a new version with the appropriate values.
 </Admonition>
 
+### Autoscaling
+
+In order to reduce DynamoDB costs you might want to enable DynamoDB autoscaling.
+This can be done manually or delegated to Teleport.
+
+This is step is usually done after a successful Teleport deployment, once you have gathered some data
+about Teleport's DynamoDB usage and know what regular usage looks like and how autoscaling should be tuned.
+See [the DynamoDB autoscaling section](../reference/backends.mdx#dynamodb-autoscaling) for more details.
+
+To go further you have to know the desired read/write minimum, maximum and target capacity.
+If you decide to let Teleport set up the autoscaling, proceed as follows:
+
+<Tabs>
+  <TabItem label="Using values.yaml">
+  Set the following fields in your existing `aws-values.yaml` file and replace the numeric values with yours:
+
+  ```yaml
+  aws:
+    # [...] already present values under `aws`
+    dynamoAutoScaling: true
+    readMinCapacity: 5     # integer
+    readMaxCapacity: 100   # integer
+    readTargetValue: 50.0  # float
+    writeMinCapacity: 5    # integer
+    writeMaxCapacity: 100  # integer
+    writeTargetValue: 50.0 # float
+
+  Then perform [a cluster upgrade](#upgrading-the-cluster-after-deployment) with the new values.
```

Comment:
If this is only the `helm upgrade` command, I think we can just include the command snippet here as well (rather than use a link) so the reader can see all of these instructions in one place.

Reply:
Done